geomstats/backend/pytorch.py

```diff
@@ -352,8 +379,8 @@ def diagonal(*args, **kwargs):
     return torch.diagonal(*args, **kwargs)
 
 
-def exp(*args, **kwargs):
-    return torch.exp(*args, **kwargs)
+def exp(input):
+    return torch.exp(torch.tensor(input))
```

Comment:
Let's put the conversion to tensor outside of this function (maybe in the core code). As we do not perform such conversions for the other functions for now, it would keep the code consistent.

Reply:
Done.